.gitlint

```diff
@@ -4,7 +4,10 @@ ignore=title-trailing-punctuation, T3, title-max-length, T1, body-hard-tab, B3,
 # verbosity should be a value between 1 and 3, the commandline -v flags take precedence over this
 verbosity = 3
 # By default gitlint will ignore merge commits. Set to 'false' to disable.
-ignore-merge-commits=true
+ignore-merge-commits=false
+ignore-revert-commits=false
```

Comment:
are you sure we want to prohibit revert commits ?

Personally I believe it to be nice when a revert commit can be easily identified, else we should establish a clear guideline as to how a pure revert commit must look.

Reply:
I am not prohibiting, I am just not ignoring them. They will still work with the new rules, as long as they are formatted as usual (output of `git revert` + `Signed-off-by` line).

Reply:
ahh, so even though git lint will fail we will just allow merge on red on case-by-case basis then, I assume.